random.js

```diff
@@ -0,0 +1,76 @@
+var slideIndex = 0;
```

Comment:
`var` is a dirty word. I see that you already use `let` so replace the rest of those `var` with `let` and `const`